ansible_rulebook/app.py

```diff
@@ -245,9 +245,12 @@ def validate_actions(startup_args: StartupArgs) -> None:
                     )
 
 
-def set_controller_params(startup_args: StartupArgs) -> None:
+async def validate_controller_params(startup_args: StartupArgs) -> None:
```

Comment:
I think I have found duplicated code: 
https://github.com/ansible/ansible-rulebook/blob/main/ansible_rulebook/cli.py#L227

Reply:
@Alex-Izquierdo Thats a different issue we need to ensure that we have dependent arguments on the command line, if you give us the controller_url you also need to give us the token. I had tried using the parser with dependent arguments so we can say if arg1 is provided we need arg2 too. When arg2 is missing we want to exit the process. Since I couldn't implement the parser we have to have that code to check the dependent arguments.